### ISSUE TITLE
Add check for binary files (and block them)

### ISSUE THREAD

**Issue by pverscha** [unipept/unipept-desktop#128]
Some users are apparently loading in binary peptide files in Unipept Desktop, which causes our backend to crash. The desktop app should check if a given file is text-based before accepting it.

**Comment by ninewise** [comment]
Could these be compressed files?

**Comment by pverscha** [comment]
Yes, these could also be compressed files. We should, nonetheless, inform the user that he's using a wrong type of input file. I'll try to make the error message as clear as possible in order not to confuse our users :)
